webui/src/components/BugTitleForm/BugTitleForm.tsx

```diff
@@ -191,7 +173,7 @@ function BugTitleForm({ bug }: Props) {
       <div className="classes.headerSubtitle">
         <Typography color={'textSecondary'}>
           <Author author={bug.author} />
-          {' opened this bug '}
+          {' opened this issue '}
```

Comment:
For better or worse, we use "bug" in all git-bug, so let's stick with that for now.

Reply:
Reverted.